### PR TITLE
separate metric checking into non zero and can-be-zero metrics

### PR DESCRIPTION
**Pull request by rootfs**
Ensure some (i.e. uncore) metrics can be ignored while useful metrics should always be valid.

### CODE REVIEW

**Comment by rootfs** [comment]
the energy metrics is still zero, i think we need to run some stress-ng workload during the test.

File: e2e/integration-test/e2e_metric_test.go [hunk 1]
```diff
@@ -260,27 +271,39 @@ var _ = Describe("Metrics check should pass", Ordered, func() {
...
 	})
 
-	_ = DescribeTable("Check node level metrics for details",
+	_ = DescribeTable("Check node level metrics for details, non zero value metric should be found",
 		func(metricName string) {
-			checkMetricValues(keplerMetric, metricName)
+			noneZeroMetricValues(keplerMetric, metricName)
+		},
+		Entry(nil, "kepler_exporter_build_info"),       // only one
+		Entry(nil, "kepler_node_core_joules_total"),    // node level
```
Comment (marceloamaral) [inline e2e/integration-test/e2e_metric_test.go:279]: The test is failing for `kepler_node_core_joules_total`. This metric is expected to be 0 right?

Comment (rootfs) [reply e2e/integration-test/e2e_metric_test.go:279]: only node platform and uncore are expected to be 0

Comment (jichenjc) [reply e2e/integration-test/e2e_metric_test.go:293]: my env shows non zero, is it because something in CI env?

Comment (rootfs) [reply e2e/integration-test/e2e_metric_test.go:293]: not sure what happened to the CI, it is always 0

Comment (rootfs) [reply e2e/integration-test/e2e_metric_test.go:291]: fixed

**Comment by rootfs** [comment]
the stress test option will be enabled in a separate PR

**Review by vprashar2929 — APPROVED** [review]
LGTM

**Pull request merged**